src/components/Alert/Alert.tsx

```diff
@@ -2,6 +2,7 @@ import * as classNames from 'classnames';
 import * as React from 'react';
 
 import * as Rivet from '../util/Rivet';
+import * as svg from '../util/RivetIcons';
```

Comment:
Is there a reason to `import *` instead of just importing the icon you need? All of the imports seem to be this way, so I'm not sure if there's a typescript-y reason for it.

Reply:
I'm changing them all to SFCs which will be imported singly. Stay tuned.